packages/components/pages/f-account-info/test/component/f-account-info.component.spec.js

```diff
@@ -4,6 +4,8 @@ const AccountInfo = require('../../test-utils/component-objects/f-account-info.c
 
 let accountInfo;
 
+const input = '123';
```

Comment:
Nit: Might be worth renaming this to `illegalInput` so it's a bit more descriptive.

Reply:
Also, can all these tests be condensed into a `foreEach`?

Reply:
ah yep they can 🤦